pages/index.js

```diff
@@ -47,33 +46,6 @@ class App extends React.Component {
             ul li, ol li {
               line-height:140%; 
             }
-            table {
-                border-collapse: collapse;
-                background:rgba(255,255,255,0.4);
-            }
-
-            th,
-            td {
-                border: 1px solid #000;
-                padding: 0.75rem;
-                text-align: left;
-            }
-
-            th {
-                font-weight: bold;
-                white-space: nowrap;
-                background: #000;
-                color: #fff;
-            }
-
-            tr:first-of-type th:not(:last-child) {
-                border-right-color: transparent;
-            }
-
-            tr:first-child th:first-child,
-            tr:not(:first-child):not(:last-child) th {
-                border-bottom-color: transparent !important;
-            }
```

Comment:
Moved table styling to the live editor because this is now part of usage.

Reply:
I prefer this level of transparency all in the live editor. Very nice.